### ISSUE TITLE
keep this github?

### ISSUE THREAD

**Issue by erikdubois** [ARCHLabs/ALOB#1]

**Comment by Dobbie03** [comment]
Yes.  This gives you the ability to back up your configurations.

**Comment by erikdubois** [comment]
online ??

**Comment by erikdubois** [comment]
i know what it does but do we need the github space?

**Comment by Dobbie03** [comment]
No locally.

**Comment by erikdubois** [comment]
delete ?

**Comment by Dobbie03** [comment]
Where would we merge it?  If you can merge it in a good repo then go for it.

**Comment by erikdubois** [comment]
i like to run a tight ship... sorry

**Comment by Dobbie03** [comment]
No apologies required at all.  Once I get the hang of GitHub, I too will be streamlining everything.

GitHub has so much to take in and learn.

**Comment by erikdubois** [comment]
i would delete it now and create it if we need it again

**Comment by Dobbie03** [comment]
Delete away my friend.
